test/functional/test_framework/messages.py

```
@@ -2563,35 +2563,45 @@ def __repr__(self):
         r += struct.pack("<B", self.quorum_type)
         r += ser_uint256(self.quorum_hash)
         r += struct.pack("<H", self.data_mask)
         r += ser_uint256(self.protx_hash)
+        if self.error is not None:
+            r += struct.pack("<B", self.error)
         return r
 
     def __repr__(self):
-        return "msg_qgetdata(quorum_hash=%064x, quorum_type=%d, data_mask=%d, protx_hash=%064x)" % (
+        return "msg_qgetdata(quorum_hash=%064x, quorum_type=%d, data_mask=%d, protx_hash=%064x, error=%s)" % (
                                                                                 self.quorum_hash,
```

Flake8 (7.3.0)

[error] continuation line over-indented for hanging indent

(E126)